backend/src/routes/api/modelRegistryRoleBindings/modelRegistryRolebindingsUtils.ts

```diff
@@ -0,0 +1,55 @@
+import { K8sStatus, KnownLabels, KubeFastifyInstance, RoleBindingKind } from '../../../types';
+import { V1RoleBinding } from '@kubernetes/client-node';
+import { IncomingMessage } from 'http';
+
+const MODEL_REGISTRY_ROLE_BINDING_API_GROUP = 'rbac.authorization.k8s.io';
+const MODEL_REGISTRY_ROLE_BINDING_API_VERSION = 'v1';
+const MODEL_REGISTRY_ROLE_BINDING_PLURAL = 'rolebindings';
+
+export const listModelRegistryRoleBindings = async (
+  fastify: KubeFastifyInstance,
+  mrNamespace: string,
+): Promise<{ items: RoleBindingKind[] }> => {
+  const response = await (fastify.kube.customObjectsApi.listNamespacedCustomObject(
+    MODEL_REGISTRY_ROLE_BINDING_API_GROUP,
+    MODEL_REGISTRY_ROLE_BINDING_API_VERSION,
+    mrNamespace,
+    MODEL_REGISTRY_ROLE_BINDING_PLURAL,
+    undefined,
+    undefined,
+    undefined,
+    KnownLabels.LABEL_SELECTOR_MODEL_REGISTRY,
+  ) as Promise<{ body: { items: RoleBindingKind[] } }>);
+  return response.body;
+};
+
+export const createModelRegistryRoleBinding = async (
+  fastify: KubeFastifyInstance,
+  rbRequest: V1RoleBinding,
+  mrNamespace: string,
+): Promise<RoleBindingKind> => {
+  const response = await (fastify.kube.customObjectsApi.createNamespacedCustomObject(
+    MODEL_REGISTRY_ROLE_BINDING_API_GROUP,
+    MODEL_REGISTRY_ROLE_BINDING_API_VERSION,
+    mrNamespace,
+    MODEL_REGISTRY_ROLE_BINDING_PLURAL,
+    rbRequest,
+  ) as Promise<{ body: RoleBindingKind }>);
+  return response.body;
+};
+
+export const deleteModelRegistriesRolebinding = async (
+  fastify: KubeFastifyInstance,
+  roleBindingName: string,
+  mrNamespace: string,
+): Promise<K8sStatus> => {
+  const { body } = (await fastify.kube.customObjectsApi.deleteNamespacedCustomObject(
+    MODEL_REGISTRY_ROLE_BINDING_API_GROUP,
+    MODEL_REGISTRY_ROLE_BINDING_API_VERSION,
+    mrNamespace,
+    MODEL_REGISTRY_ROLE_BINDING_PLURAL,
+    roleBindingName,
+  )) as { response: IncomingMessage; body: K8sStatus };
```

Comment:
Can we make the pattern consistent with the other two functions?
```ts
const response = await (/*...*/) as Promise<{ body: K8sStatus }>;
return response.body
```
Then you also don't have to import `IncomingMessage`